.github/ISSUE_TEMPLATE/credentials-access.md

```diff
@@ -0,0 +1,25 @@
+--
+name: Credential Access Request
```

Comment:
I noticed you generalized and didn't mention 1Password anywhere, wondering if that's needed? e.g. maybe

```suggestion
name: 1Password Vault Membership Request
```

Reply:
Yeah i intentionally left it off in case we change it in the future 🤷🏻

Reply:
```suggestion
name: Credential Store Access Request
```